aiokafka/consumer/assignors.py

```diff
@@ -0,0 +1,23 @@
+from kafka.coordinator.assignors.abstract import AbstractPartitionAssignor
+import abc
+
+
+class AbstractStaticPartitionAssignor(AbstractPartitionAssignor):
+    """
+    Abstract assignor implementation that also supports static assignments (KIP-345)
+    """
+
+    @abc.abstractmethod
+    def assign(self, cluster, members, member_group_instance_ids):
```

Comment:
`def assign(self, cluster, members, member_group_instance_ids): # lgtm[py/inheritance/signature-mismatch]` 

..to inform LGTM that you do really mean to have a signature mismatch.

https://lgtm.com/help/lgtm/alert-suppression#python